linkerd/core/src/main/scala/io/buoyant/linkerd/FailureAccrualInitializer.scala

```diff
@@ -0,0 +1,37 @@
+package io.buoyant.linkerd
+
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonTypeInfo}
+import com.twitter.conversions.time._
+import com.twitter.finagle.service.{Backoff, FailureAccrualFactory}
+import com.twitter.finagle.service.exp.FailureAccrualPolicy
+import com.twitter.util.Duration
+import io.buoyant.config.ConfigInitializer
+
+abstract class FailureAccrualInitializer extends ConfigInitializer
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "kind")
+trait FailureAccrualConfig {
+  @JsonIgnore
+  def policy: () => FailureAccrualPolicy
```

Comment:
This type seemed odd to me so I dug into FailureAccrualFactory to figure out why.  It turns out that the FailureAccrualPolicy isn't just a policy but also keeps the failure accrual state, which is why you would want to be able to create new ones on demand, which explains this type.  ☄️ :star: